WTWSMS/common/traits/02_traits.txt

```diff
@@ -1093,10 +1095,15 @@ schemer = {
 theologian = {
 	lifestyle = yes
 	
-	learning = 2
-	intrigue = 1
+	learning = 4
+	diplomacy = 2
```

Comment:
It's not the "only favorable trait", I wanted Scholar to have the same boost, but without the AI_ZEAL. I realize I forgot to do it to Scholar.

As for Mystic, the reason Mystic isn't a Lifestyle is because there are numerous characters and religions that automatically instate Mysticism on characters because they were highly mystical religions. (In particular, Tzadikites and Melchisedechians considered what the game calls mysticism to be core acts of worship.)

Removing it as a lifestyle allows those characters, and everyone else, to take other paths in life.